scripts/generate/configs/headersmore_config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "directiveMapName":"headersMoreDirectives",
+    "matchFuncName":"MatchHeadersMoreLatest",
+    "matchFuncComment":"MatchHeadersMoreLatest is a MatchFunc for the latest version of headersmore."
+}
```

Comment:
Nit: new line at the end of these config files please?

Reply:
Oh sure! Applied